src/session/content/message_row/bubble.rs

```diff
@@ -101,6 +106,9 @@ mod imp {
                     glib::ParamSpecObject::builder("prefix", gtk::Widget::static_type())
                         .flags(glib::ParamFlags::WRITABLE)
                         .build(),
+                    glib::ParamSpecObject::builder("reply", MessageReply::static_type())
+                        .flags(glib::ParamFlags::READWRITE | glib::ParamFlags::EXPLICIT_NOTIFY)
+                        .build(),
```

Comment:
I suggest not exposing a "reply" property since you can retrieve the reply from the "update_from_message" method and manage the MessageReply internally, which is cleaner